Failed the `dotnet pack` command in turorial.

@hskwakr commented on [Sat Dec 07 2019](https://github.com/dotnet/docs.ja-jp/issues/937)

[フィードバックをこちらに入力してください]

## Introduction  

Please let me know if this contents is not an appropriate issue.  

Please let me know if there is any missing information in this message.
Also, please let me know if you have any helpful information on this issue.

## Issue  

I tried to this tutorial:  
https://docs.microsoft.com/ja-jp/dotnet/core/tutorials/cli-templates-create-template-pack

And I failed the `dotnet pack` command:  

```powershell
PS C:\...\working\templates> dotnet pack  
.NET Core 向け Microsoft (R) Build Engine バージョン 16.4.0+e901037fe  
Copyright (C) Microsoft Corporation.All rights reserved.  

C:\...\working\templates\templatepack.csproj : error NU1100: '.NETStandard,Version=v2.0' に対する 'NETStandard.Library (>= 2.0.3)' を解決できません 。  
C:\...\working\templates\templatepack.csproj の復元が 163.08 ms で失敗しました。  
```

I learned that `.NET Core 3.1` can use `.NET Standard 2.1`.  
https://docs.microsoft.com/ja-jp/dotnet/standard/net-standard#net-implementation-support

So I changed `netstandard2.0` to ` netstandard2.1` in `templatepack.csproj`.  
And I failed the `dotnet pack` command:  

```powershell
PS C:\...\working\templates> dotnet pack  
.NET Core 向け Microsoft (R) Build Engine バージョン 16.4.0+e901037fe  
Copyright (C) Microsoft Corporation.All rights reserved.  

C:\...\working\templates\templatepack.csproj の復元が 47.55 ms で完了しました。  
templatepack -> C:\...\working\templates\bin\Debug\netstandard2.1\templatepack.dll  
C:\Program Files\dotnet\sdk\3.1.100\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(198,5): error NU5017: 依存関係とコンテンツのないパッケージは作成できません。 [C:\...\working\templates\templatepack.csproj]  
```

## MEMO

I pushed my code to github:  
https://github.com/hskwakr/dotnetTutorials  

My development environment is this:  

```
Visual Studio (VSCode)  
Version: 1.40.2 (user setup)  
Commit: f359dd69833dd8800b54d458f6d37ab7c78df520  
Date: 2019-11-25T14:54:45.096Z  
Electron: 6.1.5  
Chrome: 76.0.3809.146  
Node.js: 12.4.0  
V8: 7.6.303.31-electron.0  
OS: Windows_NT x64 10.0.18362  
```

```
dotnet --info  
.NET Core SDK (global.json を反映):  
Version:   3.1.100  
Commit:    cd82f021f4  
ランタイム環境:  
OS Name:     Windows  
OS Version:  10.0.18362  
OS Platform: Windows  
RID:         win10-x64  
Base Path:   C:\Program Files\dotnet\sdk\3.1.100\  
```

Thank you for reading.
---
#### ドキュメントの詳細

⚠ *このセクションを編集しないでください。 docs.microsoft.com で必須です ➟ GitHub の問題のリンク。*

* ID: 64d0e779-2a93-d2b8-3fb0-8c88812a4fc7
* Version Independent ID: 2e4ef5da-1b4b-a1d7-7ec8-6169360bf315
* Content: [dotnet new 用のテンプレート パックを作成する - .NET Core](https://docs.microsoft.com/ja-jp/dotnet/core/tutorials/cli-templates-create-template-pack#feedback)
* Content Source: [docs/core/tutorials/cli-templates-create-template-pack.md](https://github.com/dotnet/docs.ja-jp/blob/live/docs/core/tutorials/cli-templates-create-template-pack.md)
* Product: **dotnet-core**
* GitHub Login: @Thraka
* Microsoft Alias: **adegeo**

---

@srvbpigh commented on [Sat Dec 07 2019](https://github.com/dotnet/docs.ja-jp/issues/937#issuecomment-562846105)

Hello, @hskwakr
 
Thank you for your feedback.
 
We are actively reviewing your comments and will get back to you soon.


Kind regards,
Microsoft DOCS International Team

--- a/docs/core/tutorials/cli-templates-create-item-template.md
+++ b/docs/core/tutorials/cli-templates-create-item-template.md
@@ -28,7 +28,7 @@ In this part of the series, you'll learn how to:
 
   The reference article explains the basics about templates and how they're put together. Some of this information will be reiterated here.
 
-* Open a terminal and navigate to the _working\templates\\_ folder.
+* Open a terminal and navigate to the _working\templates_ folder.
 
 ## Create the required folders
 
@@ -49,7 +49,7 @@ parent_folder
 
 An item template is a specific type of template that contains one or more files. These types of templates are useful when you want to generate something like a config, code, or solution file. In this example, you'll create a class that adds an extension method to the string type.
 
-In your terminal, navigate to the _working\templates\\_ folder and create a new subfolder named _extensions_. Enter the folder.
+In your terminal, navigate to the _working\templates_ folder and create a new subfolder named _extensions_. Enter the folder.
 
 ```console
 working
@@ -80,7 +80,7 @@ Now that you have the content of the template created, you need to create the te
 
 ## Create the template config
 
-Templates are recognized in .NET Core by a special folder and config file that exist at the root of your template. In this tutorial, your template folder is located at _working\templates\extensions\\_.
+Templates are recognized in .NET Core by a special folder and config file that exist at the root of your template. In this tutorial, your template folder is located at _working\templates\extensions_.
 
 When you create a template, all files and folders in the template folder are included as part of the template except for the special config folder. This config folder is named _.template.config_.
 

--- a/docs/core/tutorials/cli-templates-create-project-template.md
+++ b/docs/core/tutorials/cli-templates-create-project-template.md
@@ -24,13 +24,13 @@ In this part of the series you'll learn how to:
 ## Prerequisites
 
 * Complete [part 1](cli-templates-create-item-template.md) of this tutorial series.
-* Open a terminal and navigate to the _working\templates\\_ folder.
+* Open a terminal and navigate to the _working\templates_ folder.
 
 ## Create a project template
 
 Project templates produce ready-to-run projects that make it easy for users to start with a working set of code. .NET Core includes a few project templates such as a console application or a class library. In this example, you'll create a new console project that enables C# 8.0 and produces an `async main` entry point.
 
-In your terminal, navigate to the _working\templates\\_ folder and create a new subfolder named _consoleasync_. Enter the subfolder and run `dotnet new console` to generate the standard console application. You'll be editing the files produced by this template to create a new template.
+In your terminal, navigate to the _working\templates_ folder and create a new subfolder named _consoleasync_. Enter the subfolder and run `dotnet new console` to generate the standard console application. You'll be editing the files produced by this template to create a new template.
 
 ```console
 working
@@ -93,7 +93,7 @@ Now that you have the content of the template created, you need to create the te
 
 ## Create the template config
 
-Templates are recognized in .NET Core by a special folder and config file that exist at the root of your template. In this tutorial, your template folder is located at _working\templates\consoleasync\\_.
+Templates are recognized in .NET Core by a special folder and config file that exist at the root of your template. In this tutorial, your template folder is located at _working\templates\consoleasync_.
 
 When you create a template, all files and folders in the template folder are included as part of the template except for the special config folder. This config folder is named _.template.config_.
 

--- a/docs/core/tutorials/cli-templates-create-template-pack.md
+++ b/docs/core/tutorials/cli-templates-create-template-pack.md
@@ -2,7 +2,7 @@
 title: Create a template pack for dotnet new
 description: Learn how to create a csproj file that will build a template pack for the dotnet new command.
 author: thraka
-ms.date: 06/25/2019
+ms.date: 12/10/2019
 ms.topic: tutorial
 ms.author: adegeo
 ---
@@ -24,9 +24,9 @@ In this part of the series you'll learn how to:
 
 * Complete [part 1](cli-templates-create-item-template.md) and [part 2](cli-templates-create-project-template.md) of this tutorial series.
 
-  This tutorial uses the two templates created in the first two parts of this tutorial. It's possible you can use a different template as long as you copy the template as a folder into the _working\templates\\_ folder.
+  This tutorial uses the two templates created in the first two parts of this tutorial. You can use a different template as long as you copy the template, as a folder, into the _working\templates\\_ folder.
 
-* Open a terminal and navigate to the _working\templates\\_ folder.
+* Open a terminal and navigate to the _working\\_ folder.
 
 ## Create a template pack project
 

